### DISCUSSION TITLE
Post carousel/sidecar

### DISCUSSION THREAD

**Discussion by TheForgotten69** [sbertix/Swiftagram#121] [Q&A] [answered]
Hey there!
I really like what you've done with your library @sbertix! I'm trying to do the same in golang and I am currently stuck on how to implement sidecar with the latest Instagram version. Does anyone have some insight into how these endpoints work? 
I can provide more if needed

**Comment by sbertix** [comment]
Hey @TheForgotten69, 

Sorry for not being able to reply earlier. 
I'm assuming you're referring to the "carousel"/"album" feature, as they used several different names in production, branding and behind the scenes, too. Correct? 

**Swiftagram** does not support the feature, _yet_. _Yet_ as in "it hasn't been left out on purpose and it might eventually be added", I just never felt the amount of effort and time required to have it actually work was worth it. 
So there's not much I can do to help unfortunately. 

Good luck with your project 💪

**Comment by TheForgotten69** [comment, marked as answer]
Hey @sbertix and thank you for your answer!

You were correct about the feature and I managed to make it work with the latest Twitter APK Version.

Just in case you, or anyone passing by there wants to know:

- there is a "isCarousel=1" to send when uploading media.
- All uploading data must go to a **ChildrenMetadata** object and you must generate a new upload_id and carousel_id (they are identical)
